docs/DeveloperGuide.adoc

```diff
@@ -66,3 +67,7 @@ OSes. Convert the `actual.txt` to the format used by your OS using some https://
 * Problem: Test fails during the very first time. +
 Solution: The output of the very first test run could be slightly different because the program
 creates a new storage file. Tests should pass from the 2nd run onwards.
+
+* Problem: Marked directory, `Sources Root`, keep getting unmarked. +
+Solution: This can happen if you've accidentally import the gradle library. +
+Please try to disable `"Use auto-import"` in `Settings (Preferences)` > `Build...` > `Build Tools` > `Gradle`.
```

Comment:
At the end, remind the student to re-mark the `src/` as "Source Root" after disabling Gradle.